src/nuclide.cpp

```diff
@@ -361,6 +404,142 @@ double Nuclide::elastic_xs_0K(double E) const
   return (1.0 - f)*elastic_0K_[i_grid] + f*elastic_0K_[i_grid + 1];
 }
 
+void Nuclide::calculate_urr_xs(const int i_temp, const double E)
+{
+  auto& micro = simulation::micro_xs[i_nuclide_];
+  micro.use_ptable = true;
+
+  // Create a shorthand for the URR data
+  UrrData* urr = &(urr_data_[i_temp]);
```

Comment:
A constant reference would be just as good:
```C++
const auto& urr = urr_data_[i_temp];
```
All the `urr->` below would become `urr.`